### DISCUSSION TITLE
Strange Segmentation output. Any idea what's happening here?

### DISCUSSION THREAD

**Discussion by che85** [Project-MONAI/MONAILabel#1204] [Q&A] [answered]
Hello, 

I was wondering if anyone has a sense of what's going on here. When running my model locally, I am getting three segments that make sense, but when using the MONAILabel client, I am getting the following result.

<img width="1218" alt="image" src="https://user-images.githubusercontent.com/10195822/207104065-d82d6c9c-6c2e-47ea-bc7a-d5ef538feafe.png">

**Comment by che85** [comment]
@diazandr3s @SachidanandAlle Any ideas? Maybe you have seen this kind of behavior before?

**Comment by tangy5** [comment]
Hi @che85 , thanks for reporting this. 
May I ask if you used the post transform of "FindContour" in your application and code?
This code had recent changes with a moveaxis, we can double check if it's related to this updates. 

Thank you.

**Comment by SachidanandAlle** [comment, marked as answer]
to slice and dice the issue.. you can run the segmentation without 3D Slicer.. by directly using rest apis (http://127.0.0.1:8000) and try /infer API.. supply image id and model and select image only in the response type.. if the output segments are correct then need to dig into 3d slicer.. if the output itself is not correct.. then you can dig more into inference workflow.. each transform and infer output is making sense or not..

**Comment by che85** [comment]
@SachidanandAlle Thank you. Your tips were really helpful. I was not aware that I could ask the rest api only to return the image for debugging. It was indeed an issue with my pre_transforms.
